app/helpers/spree/frontend_helper_decorator.rb

```diff
@@ -1,50 +1,23 @@
-        ) << content_tag(:span, nil, itemprop: 'item', itemscope: 'itemscope', itemtype: 'https://schema.org/Thing', itemid: spree.root_path), itemscope: 'itemscope', itemtype: 'https://schema.org/ListItem', itemprop: 'itemListElement', class: 'breadcrumb-item')]
-  
-        if taxon
-          ancestors = taxon.ancestors.where.not(parent_id: nil)
-  
-          # breadcrumbs for ancestor taxons
-          crumbs << ancestors.each_with_index.map do |ancestor, index|
-            content_tag(:li, content_tag(
-              :a, content_tag(
-                :span, ancestor.name, itemprop: 'name'
-              ) << content_tag(:meta, nil, itemprop: 'position', content: index + 1), itemprop: 'url', href: seo_url(ancestor, params: permitted_product_params)
-            ) << content_tag(:span, nil, itemprop: 'item', itemscope: 'itemscope', itemtype: 'https://schema.org/Thing', itemid: seo_url(ancestor, params: permitted_product_params)), itemscope: 'itemscope', itemtype: 'https://schema.org/ListItem', itemprop: 'itemListElement', class: 'breadcrumb-item')
-          end
-  
-          # breadcrumbs for current taxon
-          crumbs << content_tag(:li, content_tag(
-              :div,content_tag(
-              :span, taxon.name, itemprop: 'name'
-            ) << content_tag(:meta, nil, itemprop: 'position', content: ancestors.size + 1), itemprop: 'url', href: seo_url(taxon, params: permitted_product_params)
-          ) << content_tag(:span, nil, itemprop: 'item', itemscope: 'itemscope', itemtype: 'https://schema.org/Thing', itemid: seo_url(taxon, params: permitted_product_params)), itemscope: 'itemscope', itemtype: 'https://schema.org/ListItem', itemprop: 'itemListElement', class: 'breadcrumb-item active' )
-  
-          # breadcrumbs for product
-          if product
-            crumbs << content_tag(:li, content_tag(
-              :span, content_tag(
-                :span, product.name, itemprop: 'name'
-              ) << content_tag(:meta, nil, itemprop: 'position', content: ancestors.size + 2), itemprop: 'url', href: spree.product_path(product, taxon_id: taxon&.id)
-            ) << content_tag(:span, nil, itemprop: 'item', itemscope: 'itemscope', itemtype: 'https://schema.org/Thing', itemid: spree.product_path(product, taxon_id: taxon&.id)), itemscope: 'itemscope', itemtype: 'https://schema.org/ListItem', itemprop: 'itemListElement', class: 'breadcrumb-item ')
+    return '' if max_level < 1
+    selected_parent_taxon_name = params["id"]
+    show_klass =  (root_taxon.children.pluck("permalink").include?(selected_parent_taxon_name) && root_taxon.parent.present?) ? 'show' : ''
+    parent_klass = (root_taxon.children.present? && ((root_taxon&.parent&.name == "Categories") || (root_taxon == current_taxon ))) ? 'sidebar-sub-categories' : "dropdown-menu sub-child-manu-js #{show_klass}"
+    content_tag :ul, class: parent_klass   do
+      
+      taxons = root_taxon.children.map do |taxon|
+        if taxon.products.present?
```

Comment:
@Umer, why we added products presence check. I think its not needed. The taxon name should show if taxon dnt have any product. Please discuss this use case with me.

Reply:
this case is handle in bestflavor that is why i added it here